<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastro_entidade</class>
 <widget class="QMainWindow" name="cadastro_entidade">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Entidades</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_entidades">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>267</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab_lista">
     <attribute name="title">
      <string>Lista</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>-6</x>
        <y>9</y>
        <width>397</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Nome</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_pesquisar_nome">
         <property name="inputMask">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_pesquisar">
         <property name="text">
          <string>Pesquisar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="tableWidget_entidades">
      <property name="geometry">
       <rect>
        <x>-2</x>
        <y>46</y>
        <width>401</width>
        <height>195</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>6</number>
      </property>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_cadastro">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Cadastro</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_id_1">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>11</y>
        <width>37</width>
        <height>24</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>9999</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_fornecedor">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>12</y>
        <width>85</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Fornecedor</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_funcionario">
      <property name="geometry">
       <rect>
        <x>151</x>
        <y>12</y>
        <width>85</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Funcion√°rio</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>251</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Nome *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_nome">
         <property name="maxLength">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>121</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_35">
         <property name="text">
          <string>Telefone *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_telefone">
         <property name="inputMask">
          <string>+99 999 99999-9999</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>110</y>
        <width>251</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Email *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_email">
         <property name="maxLength">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>110</y>
        <width>121</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>CPF *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_cpf">
         <property name="inputMask">
          <string>999.999.999-99</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>121</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Data nascimento</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_data_nascimento">
         <property name="inputMask">
          <string>99/99/9999</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>121</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Data cadastro</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_data_cadastro">
         <property name="inputMask">
          <string>99/99/9999</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>170</y>
        <width>121</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>RG *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_rg">
         <property name="inputMask">
          <string>9999999999999</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_fornecedor">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>Fornecedor</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_id_fornecedor">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>11</y>
        <width>37</width>
        <height>24</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>9999</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>110</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Cod. Interno</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_codigo_interno">
         <property name="inputMask">
          <string>9999</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>371</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Nome fantasia *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_nome_fantasia">
         <property name="maxLength">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>371</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Razao social *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_razao_social">
         <property name="maxLength">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>170</y>
        <width>120</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>CNPJ *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_cnpj">
         <property name="inputMask">
          <string>99.999.999/9999-99</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>170</y>
        <width>120</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Inscri√ß√£o estadual *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_inscricao_estadual">
         <property name="inputMask">
          <string>9999999999</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_funcionario">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>Funcion√°rio</string>
     </attribute>
     <widget class="QRadioButton" name="radioButton_ativo">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>11</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Ativo</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_funcionario">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>11</y>
        <width>37</width>
        <height>24</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>9999</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>110</y>
        <width>130</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <widget class="QLabel" name="label_34">
         <property name="text">
          <string>Data recis√£o</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_data_recisao">
         <property name="inputMask">
          <string>99/99/9999</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>120</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <widget class="QLabel" name="label_33">
         <property name="text">
          <string>Data admiss√£o *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_data_admissao">
         <property name="inputMask">
          <string>99/99/9999</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>100</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <widget class="QLabel" name="label_32">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Permiss√µes *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_permissoes">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="inputMask">
          <string>AAA</string>
         </property>
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>100</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_16">
       <item>
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>Conta corrente *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_conta_corrente">
         <property name="inputMask">
          <string>9999 9999 9999 9999</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>120</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_17">
       <item>
        <widget class="QLabel" name="label_30">
         <property name="text">
          <string>Usu√°rio *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_usuario">
         <property name="maxLength">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>130</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_18">
       <item>
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>Senha *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_senha">
         <property name="maxLength">
          <number>100</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_endereco">
     <attribute name="title">
      <string>Endere√ßo</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_vincular_endereco">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>46</y>
        <width>380</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Vincular endere√ßo de cobran√ßa</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>67</x>
        <y>14</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Endere√ßo de cobran√ßa</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_vincular_endereco_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>122</y>
        <width>380</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Vincular endere√ßo de entrega</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_endereco_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>87</y>
        <width>37</width>
        <height>24</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>9999</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Endere√ßo de entrega</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_endereco">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>11</y>
        <width>37</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>381</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_novo_">
       <property name="text">
        <string>Novo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_gravar_">
       <property name="text">
        <string>Gravar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_remover">
       <property name="text">
        <string>Remover</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_voltar">
       <property name="text">
        <string>Voltar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_11">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>480</y>
      <width>381</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton_novo_1">
       <property name="text">
        <string>Novo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_gravar_1">
       <property name="text">
        <string>Gravar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_remover_2">
       <property name="text">
        <string>Remover</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_voltar_2">
       <property name="text">
        <string>Voltar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget_entidades</tabstop>
  <tabstop>lineEdit_pesquisar_nome</tabstop>
  <tabstop>pushButton_pesquisar</tabstop>
  <tabstop>tableWidget_entidades</tabstop>
  <tabstop>lineEdit_id_1</tabstop>
  <tabstop>checkBox_fornecedor</tabstop>
  <tabstop>checkBox_funcionario</tabstop>
  <tabstop>lineEdit_nome</tabstop>
  <tabstop>lineEdit_telefone</tabstop>
  <tabstop>lineEdit_email</tabstop>
  <tabstop>lineEdit_cpf</tabstop>
  <tabstop>lineEdit_data_cadastro</tabstop>
  <tabstop>lineEdit_data_nascimento</tabstop>
  <tabstop>lineEdit_rg</tabstop>
  <tabstop>lineEdit_id_fornecedor</tabstop>
  <tabstop>lineEdit_nome_fantasia</tabstop>
  <tabstop>lineEdit_razao_social</tabstop>
  <tabstop>lineEdit_codigo_interno</tabstop>
  <tabstop>lineEdit_cnpj</tabstop>
  <tabstop>lineEdit_inscricao_estadual</tabstop>
  <tabstop>lineEdit_id_funcionario</tabstop>
  <tabstop>radioButton_ativo</tabstop>
  <tabstop>lineEdit_conta_corrente</tabstop>
  <tabstop>lineEdit_usuario</tabstop>
  <tabstop>lineEdit_senha</tabstop>
  <tabstop>lineEdit_permissoes</tabstop>
  <tabstop>lineEdit_data_admissao</tabstop>
  <tabstop>lineEdit_data_recisao</tabstop>
  <tabstop>lineEdit_id_endereco</tabstop>
  <tabstop>pushButton_vincular_endereco</tabstop>
  <tabstop>lineEdit_id_endereco_2</tabstop>
  <tabstop>pushButton_vincular_endereco_2</tabstop>
  <tabstop>pushButton_novo_</tabstop>
  <tabstop>pushButton_gravar_</tabstop>
  <tabstop>pushButton_remover</tabstop>
  <tabstop>pushButton_voltar</tabstop>
  <tabstop>pushButton_novo_1</tabstop>
  <tabstop>pushButton_gravar_1</tabstop>
  <tabstop>pushButton_remover_2</tabstop>
  <tabstop>pushButton_voltar_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
